modules/netweaver_svpc/README.md

```diff
@@ -0,0 +1,115 @@
+
+# Netweaver  SVPC Submodule
+
+This module is handles opinionated Netweaver configuration and deployment.
+
+## Usage
+
+The resources/services/activations/deletions that this module will create/trigger are:
+
+- Create a Compute Instance that will host Netweaver
+- Create a Static IP Addresses for the Compute Instance
+- Create a 2 Persitent Disks to host Netweaver's File systems
+
+You can go in the [examples](../../examples) folder complete working example. However, here's an example of how to use the module in a main.tf file.
+
+```hcl
+provider "google" {
+  version = "~> 2.6.0"
+}
+
+module "gcp_netweaver_svpc" {
+  source                 = "terraform-google-modules/sap/google/modules/netweaver_svpc"
+  post_deployment_script = "${var.post_deployment_script}"
```

Comment:
Again, examples should be informative, not just demonstrating how to proxy every argument with a variable.

Reply:
removed these arguments.